.github/workflows/test.yml

```diff
@@ -28,12 +28,32 @@ jobs:
   test-test262:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: use node.js v16.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: 16.x
-    - run: npm ci
-    - run: npm run test262
-      env:
-        HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+      - uses: actions/checkout@v2
+      - name: use node.js v16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - run: npm ci
+      - run: npm run test262
+        env:
+          HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+  test-lint:
```

Comment:
This test-lint rule duplicates the one already in `lint.yml`, maybe test-prune should go in that file as well?

Reply:
Oops, I can't believe I didn't realize that the other yml file was where lint lived, despite its name. I assumed that test.yml was where all the checks lived!  Fixed in latest push.